### ISSUE TITLE
IMOS:  check_quality_control_variable_standard_name EXCEPTION

### ISSUE THREAD

**Issue by lbesnard** [aodn/compliance-checker#10]
```
Running Compliance Checker on the dataset from: /mnt/ebs/wip/ANMN/timeSeries/new_tb/IMOS_ANMN-NRS_CKOSTUZ_20120825T054709Z_NRSDAR_FV01_NRSDAR-1208-Surface-WQM-1_END-20121209T014808Z_C-20150324T034821Z.nc
The following exceptions occured during the imos checker (possibly indicate compliance checker issues):
imos.check_quality_control_variable_standard_name: NetCDF: Attribute not found
```

**Update 1/07/15 1120**
same with following files

```
/mnt/ebs/wip/ANMN/timeSeries/new_tb/IMOS_ANMN-NRS_KOSTUZ_20081127T161522Z_NRSESP_FV01_NRSESP-0811-WQM-21_END-20090223T100118Z_C-20150520T023848Z.nc
/mnt/ebs/wip/ANMN/timeSeries/new_tb/IMOS_ANMN-NRS_KOSTUZ_20081127T160032Z_NRSESP_FV01_NRSESP-0811-WQM-48_END-20090223T100144Z_C-20150520T025008Z.nc
/mnt/ebs/wip/ANMN/timeSeries/new_tb/IMOS_ANMN-NRS_CKOSTUZ_20140905T232944Z_NRSDAR_FV01_NRSDAR-1208-Surface-WQM-1_END-20150203T221531Z_C-20150331T061719Z.nc
/mnt/ebs/wip/ANMN/timeSeries/new_tb/IMOS_ANMN-NRS_CKOSTUZ_20140120T055944Z_NRSDAR_FV01_NRSDAR-1208-Surface-WQM-1_END-20140620T001505Z_C-20150325T235211Z.imos.check
```

**Comment by mhidas** [comment]
@fxmzb123 Could you take a look at this? I think we're not handling the case where the qc variable and the data variable don't have a standard_name attribute.

[Here](http://thredds.aodn.org.au/thredds/catalog/IMOS/eMII/checker_test/ANMN/timeSeries/catalog.html?dataset=IMOS/eMII/checker_test/ANMN/timeSeries/IMOS_ANMN-NRS_CKOSTUZ_20120825T054709Z_NRSDAR_FV01_NRSDAR-1208-Surface-WQM-1_END-20121209T014808Z_C-20150324T034821Z.nc) is the file.

**Comment by fxmzb123** [comment]
@lbesnard and @mhidas I have created a pull request to fix this.

**Comment by lbesnard** [comment]
cheers. I'll test it

**Comment by mhidas** [comment]
#25 awaiting rebase.

**Comment by mhidas** [comment]
Fixed via #25
